packages/firebase/compat/index.d.ts

```diff
@@ -2031,6 +2031,16 @@ declare namespace firebase.remoteConfig {
    * Defines levels of Remote Config logging.
    */
   export type LogLevel = 'debug' | 'error' | 'silent';
+  /**
```

Comment:
@egilmorez This is the only new block here, the rest is the automatic formatter cleaning up.

Reply:
LG!